src/Pact.hs

```diff
@@ -32,17 +33,31 @@ hasNullValue :: AT.Pair -> Bool
 hasNullValue (_, Null) = True
 hasNullValue _ = False
 
+newtype Headers = Headers [(String, String)]
```

Comment:
Beginners question: What's the advantage of using newtype and a Monoid instance here instead of a type alias?